EDI/Operation/06_ManageMappingSettings.md

```diff
@@ -0,0 +1,65 @@
+[!!Exports](../UserInterface/01_Exports.md)
+
+# Manage mapping settings
+
+The mapping settings define some basic data that is needed to be able to send EDIFACT message from *Actindo* to your business partner.
+
+## Create mapping settings
+ Create the mapping settings to define some basic data. Actindo will use it to convert the data and to automatically fill in some of the EDIFACT segments.
+
+#### Prerequisites
+At least one definition setting has been created, see [Create basic definition settings](./01_ManageDefinitions.md#create-basic-definition-settings).
+>[INFO] The *Mapping settings* tab is visible only if you have created and saved a basic definition setting. 
+
+#### Procedure
+*DataHub Exporter > Exports*
+
+![Export definitions](../../Assets/Screenshots/EDI/Operation/ExportDefinitions.png "[Export definitions]")
+
+1. Click the exporter definition for which you want to define the mapping settings.   
+    The *Edit definition "Definition name"* view is displayed. The last selected tab is displayed by default.
```

Comment:
The last selected tab is displayed by default. -> Oben: The *Settings* tab or the last selected tab is displayed...
Welche Version ist richtig? Ggf. bitte vereinheitlichen.

Reply:
done